### PR TITLE
Matt/pypi setup

### PR DESCRIPTION
**Pull request by mauerbac**
@jgaul

### CODE REVIEW

**Comment by mauerbac** [comment]
@jgaul ping

File: MANIFEST [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# file GENERATED by distutils, do NOT edit
```
Comment (jgaul) [inline MANIFEST]: this shouldn't need to be checked in; it's auto-generated when building the package, right?

File: setup.py [hunk 2]
```diff
@@ -2,10 +2,13 @@
 
 setup(name='optimizely',
       version='0.3',
-      description='An interface to Optimizely\'s REST API.',
+      description='A Python interface to Optimizely\'s REST API.',
       url='https://github.com/optimizely/optimizely-client-python',
       author='Optimizely',
+      author_email = 'developers@optimizely.com',
```
Comment (jgaul) [inline setup.py]: there shouldn't be any spaces around the `=` signs in this file

**Comment by mauerbac** [comment]
@jgaul made the two changes

**Comment by jgaul** [comment]
LGTM!

**Pull request merged**